### ISSUE TITLE
Image upload on mobile theme appear as if it is broken

### ISSUE THREAD

**Issue by DaazKu** [vanilla/vanilla#5660]
A while back we separated file upload from image upload a while back.
See https://github.com/vanilla/lithe/issues/53#issuecomment-271919976 and https://github.com/vanilla/vanilla/pull/5016

The problem is that some formats (used on mobile) do not support adding the resulting image tag in the post:
https://github.com/vanilla/vanilla/blob/bae8437ddc9dd318859afd0dc034cd77ffb855de/plugins/editor/js/editor.js#L697

So the image is uploaded but we get no feedback what so ever that this is the case.
*Note: the comment in the function linked above is confusing...*

The best possible solution would be to get the fileupload preview when buildImgTag returns nothing which would give us something like this:
![screen shot 2017-06-12 at 5 06 11 pm](https://user-images.githubusercontent.com/2412909/27055184-9faf7756-4f91-11e7-9f4b-49372cf7ddd6.png)
To get that result I hacked the imageupload to use fileupload instead.

TL;DR; imageupload is incompatible with the formats text and textex because we get no feedback that the upload was a success.
